### PR TITLE
SWAG-19535 add eslint plugin for http timeout

### PR DESCRIPTION
**Pull request by master-bogdan**

### CODE REVIEW

File: eslint-plugin-http-module-timeout/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            context.report({
+              node: node,
+              message: "HttpModule.register method call should have a timeout argument"
+            });
+          }
+        }
+      };
+    },
+  },
+  };
```
Comment (ihors) [inline eslint-plugin-http-module-timeout/index.js:19]:
```suggestion
};
```

File: eslint-plugin-http-module-timeout/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "eslint-plugin-http-module-timeout",
+    "version": "1.1.0",
+    "main": "index.js"
+  }
```
Comment (ihors) [inline eslint-plugin-http-module-timeout/package.json:5]:
```suggestion
  "name": "eslint-plugin-http-module-timeout",
  "version": "1.1.0",
  "main": "index.js"
}
```

**Review by ihors — COMMENTED** [review]
please check formatting of JS/JSON files

File: package.json [hunk 3]
```diff
@@ -36,7 +36,8 @@
     "eslint-plugin-no-only-tests": "^2.6.0",
     "eslint-plugin-prettier": "^5.0.1",
     "eslint-plugin-security": "^1.5.0",
-    "eslint-plugin-spellcheck": "^0.0.20"
+    "eslint-plugin-spellcheck": "^0.0.20",
+    "eslint-plugin-http-module-timeout": "file:eslint-plugin-http-module-timeout"
```
Comment (cyxn) [inline package.json:40]: not sure it can be peer dependency. does it actually work?

**Review by change-my-username — APPROVED** [review]

**Review by ihors — APPROVED** [review]

**Review by cyxn — APPROVED** [review]

**Pull request merged**